libs/akita/src/lib/arrayToggle.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Create an array value comparator for a specific key of the value.
+ * @param prop The property of the value to be compared.
+ */
```

Comment:
Please update the docs. Add also an example of toggling an array of primitives such as strings.